modules/drivers/druid/src/metabase/driver/druid/query_processor.clj

```diff
@@ -84,41 +89,48 @@
       :else
       (throw (Exception. "Unknown aggregation type!")))))
 
-(defmethod ->rvalue :field-id [[_ field-id]]
+(defmethod ->rvalue :field-id
+  [[_ field-id]]
   (:name (qp.store/field field-id)))
 
-(defmethod ->rvalue :datetime-field [[_ field]]
+(defmethod ->rvalue :datetime-field
+  [[_ field]]
   (->rvalue field))
 
-(defmethod ->rvalue :absolute-datetime [[_ timestamp unit]]
+(defmethod ->rvalue :absolute-datetime
+  [[_ timestamp unit]]
   (du/date->iso-8601 (du/date-trunc unit timestamp (get-timezone-id))))
 
 ;; TODO - not 100% sure how to handle times here, just treating it exactly like a date will have to do for now
-(defmethod ->rvalue :time [[_ time unit]]
+(defmethod ->rvalue :time
+  [[_ time unit]]
   (du/date->iso-8601 (du/date-trunc unit time (get-timezone-id))))
 
-(defmethod ->rvalue :relative-datetime [[_ amount unit]]
+(defmethod ->rvalue :relative-datetime
+  [[_ amount unit]]
   (du/date->iso-8601 (du/date-trunc unit (du/relative-date unit amount) (get-timezone-id))))
 
-(defmethod ->rvalue :value [[_ value]]
+(defmethod ->rvalue :value
+  [[_ value]]
```

Comment:
I really don't have any complaint about putting arg lists on the same line for functions or methods without a docstring, any reason in particular to prefer always putting them on their own line?

Reply:
Not really except for consistency's sake (and multimethods with gnarly dispatch values are perhaps easier to glance)